<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>currentprice</class>
 <widget class="QWidget" name="currentprice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>0</y>
     <width>251</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#d9d8d4  ;</string>
   </property>
   <widget class="QPushButton" name="Mywallets_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>My wallets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Transmission_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Transmission</string>
    </property>
   </widget>
   <widget class="QLabel" name="app_logo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>E:/Downloads/logo for a currency exchange named heliacal (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>-6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Dashboard_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Authentication_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: #4fee93;
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Authentication</string>
    </property>
   </widget>
   <widget class="QLabel" name="app_name">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border: none;  
color: #344b90;
background-color: #d9d8d4;
font: 20pt &quot;Rubik Wet Paint&quot;;
letter-spacing: -2px;
word-spacing: 3px;
</string>
    </property>
    <property name="text">
     <string>Spiral exchange</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Profile_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: #c97940;
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currentprice_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Current price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deposit_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>Deposit </string>
    </property>
   </widget>
   <widget class="QPushButton" name="market_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>market</string>
    </property>
   </widget>
   <widget class="QPushButton" name="withdrawal_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>750</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string> withdrawal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="easyexchange_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string> Exchange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>810</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(0, 0, 0);
font: 24pt &quot;Bangers&quot;;
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
